msg/Detection3D.msg

```diff
@@ -1,8 +1,7 @@
 # Defines a 3D detection result.
 #
 # This extends a basic 3D classification by including position information,
-#   allowing a classification result for a specific position in an image to
-#   to be located in the larger image.
+#   allowing a classification result for a specific position in data.
```

Comment:
You removed the verb ("located"). Maybe rephrase the whole comment? Something like this:

"This extends a basic 3D classification by including the pose of the detected object."

Reply:
Yes, you're right, I was also unhappy about the sentence but haven't quickly come up with anything better.